modulemd/include/private/modulemd-yaml.h

```diff
@@ -824,6 +1282,23 @@ skip_unknown_yaml (yaml_parser_t *parser, GError **error);
     }                                                                         \
   while (0)
 
+/**
+ * EMIT_STRING_SET:
+ * @emitter: (inout): A libyaml emitter object that is positioned at the start
+ * of where a new sequence will be written.
+ * @error: (out): A #GError that will return the reason for any error.
+ * @key: (in): The name to be used as the identifier for the output sequence.
+ * @table: (in): The #GHashTable that is to be output.
```

Comment:
Indicate that this `GHashTable` must be a `set()` of string keys.

Also, it outputs the keys, not the values of the table.